src/UNIV2LPOracle.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-License-Identifier: AGPL-3.0-or-later
 
 /// UNIV2LPOracle.sol
 
```

Comment:
I think we can remove old copyright comment on L5 and replace it with the following:
`// Copyright (C) 2019-2022 Dai Foundation`